dot/rpc/modules/chain_test.go

```diff
@@ -56,10 +110,10 @@ func TestChainGetHeader_Latest(t *testing.T) {
 	chain := newChainService(t)
 	svc := NewChainModule(chain.Block)
 	expected := &ChainBlockHeaderResponse{
-		ParentHash:     "0x0000000000000000000000000000000000000000000000000000000000000000",
-		Number:         big.NewInt(0),
-		StateRoot:      "0x03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c111314",
-		ExtrinsicsRoot: "0x03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c111314",
+		ParentHash:     "0xdbfdd87392d9ee52f499610582737daceecf83dc3ad7946fcadeb01c86e1ef75",
```

Comment:
maybe just a comment to explain where this and the other test hashes are coming from?